errors/http.go

```diff
@@ -0,0 +1,132 @@
+// Copyright © 2017 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+package errors
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// CodeHeader is the header where the error code will be stored
+const CodeHeader = "X-TTN-Error-Code"
+
+// HTTPStatusCode returns the corresponding http status code from an error type
+func (t Type) HTTPStatusCode() int {
```

Comment:
Make this the same as https://github.com/grpc-ecosystem/grpc-gateway/blob/7195ea445241f9773e71fc7b94c306d3ed6b35b6/runtime/errors.go#L14-L55